.travis/format-cpp.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+clang-format-6.0 --version
+
```

Comment:
Again, we should pin to the latest currently available on the BBB distribution in our base system.

Reply:
Per https://packages.ubuntu.com/search?suite=bionic&searchon=names&keywords=clang-format , it looks like Bionic (18.04) ships 6.0.

.. which is good, because Travis still has Xenial as the latest they offer and 6.0 is as high as Xenial can go without PPAs.